transformations/concept2sentence/test.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "concept2setence",
+    "test_cases": []
+}
```

Comment:
If you are skipping these test cases - which I wouldn't recommend, it is better to add them in the README for everyone's visibility.

Reply:
Added to the README